Xamarin.Essentials/Screenshot/Screenshot.android.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Android.Graphics;
+using Android.Util;
+using Android.Views;
+using Path = System.IO.Path;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        static bool PlatformCanCapture => !Platform.WindowManager.DefaultDisplay.Flags.HasFlag(DisplayFlags.Secure);
+
+        static string GetTempFileName() => Path.Combine(FileSystem.CacheDirectory, Path.GetTempFileName());
```

Comment:
@jamesmontemagno @Redth I am still not really happy with this... This folder is publicly available to the user, so _should_ we be putting things in here? @Mrnikbobjeff previously had it in the external storage folder, and I sort of felt it is better in the cache folder. I think it is better now, but maybe we need to decide on what we want to do with files we create. Should we have a special folder in the cache where we work? Should we truly use some OS temp system?